objectscale-manager/crds/storagepolicy.yaml

```
@@ -0,0 +1,25 @@
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: storagepolicies.ecs.dellemc.com
+  labels:
+    app.kubernetes.io/name: objectscale-manager
+    helm.sh/chart: objectscale-manager
+    app.kubernetes.io/component: objectscale-operator
+    app.kubernetes.io/part-of: objectscale-manager
+    app.kubernetes.io/managed-by: nautilus
+spec:
+  group: ecs.dellemc.com
+  names:
+    kind: StoragePolicy
+    listKind: StoragePolicyList
+    plural: ecs-storagepolicies
+    singular: ecs-storagepolicy
```

jason wants us to not use `ecs` in the charts.  can we use `objectscale-storagepolicies`
```suggestion
    singular: ecs-storagepolicy
```

---

fixed